testsuite/sequential_execution_performance.py

```diff
@@ -4,7 +4,7 @@
 
 # Set the tps threshold for block size 1k, 10k and 50k
 BLOCK_SIZES = ["1k", "10k", "50k"]
-THRESHOLDS = {"1k": 3000, "10k": 4000, "50k": 7000}
+THRESHOLDS = {"1k": 3200, "10k": 4300, "50k": 7400}
```

Comment:
@zekun000 @danielxiangzl : using HashMap instead of BTreeMap.. and it was always BTreeMap https://github.com/aptos-labs/aptos-core/blob/feec33fe9436a219346b5b03fbe5d32fc6200ef8/aptos-move/aptos-vm/src/data_cache.rs#L42, wonder why I didn't notice while refactoring sequential :-)